scopes/envs/envs/environments.main.runtime.ts

```diff
@@ -92,6 +92,19 @@ export class EnvsMain {
   }
 
   /**
+   * apply a list of Env Transformers on an Env, to produce a new (transformed) Env
+   */
+  transformEnv(baseEnv: Environment, envTransformers: EnvTransformer[]) {
```

Comment:
change to apply env
change base env back to target env

Reply:
done